src/AltinnCore/Templates/Dockerfile

```diff
@@ -0,0 +1,21 @@
+#altinn-runtime with sdk
+FROM altinntjenestercontainerregistry.azurecr.io/altinn-runtime:333 AS build
```

Comment:
Will this always be runtime image nr 333?
Or is this a variable that changes?

Reply:
Per now this is the only place where the Service Owner choose which Altinn-Runtime version the Service will be deployed with. In the future it would probably reference to a config file configured through Altinn Studio.